addons/website_profile/controllers/main.py

```diff
@@ -30,18 +31,10 @@ def _check_avatar_access(self, user_id, **post):
             return user.website_published and user.karma > 0
         return False
 
-    def _get_default_avatar(self, field, headers, width, height):
+    def _get_default_avatar(self):
         img_path = modules.get_module_resource('web', 'static/src/img', 'placeholder.png')
         with open(img_path, 'rb') as f:
-            image = f.read()
-        content = base64.b64encode(image)
-        dictheaders = dict(headers) if headers else {}
-        dictheaders['Content-Type'] = 'image/png'
-        if not (width or height):
-            suffix = field.split('_')[-1] if '_' in field else 'large'
-            if suffix in ('small', 'medium', 'large', 'big'):
-                content = getattr(tools, 'image_resize_image_%s' % suffix)(content)
-        return content
+            return base64.b64encode(f.read())
```

Comment:
Particular reason why is the 'resize guess based on name' got moved outside the `_get_default_avatar`?

Reply:
It was copy pasted there from another place, so I moved it to a central place to remove duplication.